base/applications/rapps/include/available.h

```diff
@@ -35,8 +50,9 @@ struct CAvailableApplicationInfo
     ATL::CStringW m_szSize;
     ATL::CStringW m_szUrlSite;
     ATL::CStringW m_szUrlDownload;
-    ATL::CStringW m_szCDPath;
     ATL::CSimpleArray<LCID> m_LanguageLCIDs;
+    ATL::CStringW m_szSnapshotFilename[MAX_SNAPSHOT_NUM];
```

Comment:
Why not use an `ATL:CSimpleArray` instead of a fixed-size array?

Reply:
I think it's almost certain that we are going to limit the maximum number of snapshots. Displaying an infinite number of screenshots makes no sense. As 16 (MAX_SNAPSHOT_NUM) is not big, I think it's ok to use a fixed-size array.

why 16? I don't know 😝 I think it's enough to store snapshots of a normal software and As a programmer, it looks more like a round number than 10 or 20, so I chose it. Anyway, it's a macro, we can change it at any time.